data/mods/Aftershock/recipes/frankenstein_recipes.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "type": "recipe",
+    "result": "inactive_frankenstein",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_CBMS",
+    "skill_used": "electronics",
+    "skills_required": [ "firstaid", 8 ],
+    "difficulty": 7,
+    "time": 50000,
+    "reversible": true,
+    "decomp_learn": 5,
+    "book_learn": [ [ "cyrus'_notes", 7 ] ],
+    "using": [ [ "soldering_standard", 20 ] ],
+    "tools": [ [ [ "afs_solderers_item", -1 ], [ "toolset", -1 ] ] ],
+    "qualities": [ { "id": "SCREW_FINE", "level": 1 } ],
+    "components": [
+      [ [ "power_supply", 6 ], [ "UPS_off", 1 ] ],
+      [ [ "amplifier", 4 ] ],
```

Comment:
Maybe use Probably use biomaterial scrap instead?

Reply:
This may sound weird but when I'm doing WIP I'd rather have PRs that show as green that I know I need to work on; than PRs that show as red that may be perfect once the PRs they are waiting on  merge ahead of them.

Reply:
Yeah seems fine to me.